### PR TITLE
Fix namespace of Sabre classes

### PR DESCRIPTION
**Pull request by nickvergessen**
- [ ] Depends on owncloud/core#12864

@DeepDiver1975 @georgehrke

### CODE REVIEW

File: appinfo/remote.php [hunk 1]
```diff
@@ -19,16 +19,16 @@
...
+$principalBackend = new OC\Connector\Sabre\Principal();
+$caldavBackend    = new OCA\Contacts\Sabre\CalDAV();
+$requestBackend = new OC\Connector\Sabre\Request();
 
 // Root nodes
 $Sabre_CalDAV_Principal_Collection = new \Sabre\CalDAV\Principal\Collection($principalBackend);
 $Sabre_CalDAV_Principal_Collection->disableListing = true; // Disable listening
 
-$calendarRoot = new OC_Connector_Sabre_CalDAV_CalendarRoot($principalBackend, $caldavBackend);
+$calendarRoot = new OCA\Contacts\Sabre\CalendarRoot($principalBackend, $caldavBackend);
```
Comment (georgehrke) [inline appinfo/remote.php]: why OCA\Contacts?

File: lib/sabre/caldav.php [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <?php
 
-class OC_Connector_Sabre_CalDAV extends \Sabre\CalDAV\Backend\AbstractBackend {
+namespace OCA\Contacts\Sabre;
```
Comment (georgehrke) [inline lib/sabre/caldav.php]: the same here

File: appinfo/remote.php (hunk 1 above)
Comment (nickvergessen) [inline appinfo/remote.php]: Because that is where the app belongs to

Comment (LukasReschke) [inline appinfo/remote.php]: The calendar app belongs to `OCA\Contacts`?

Comment (nickvergessen) [inline appinfo/remote.php]: ah sorry now I see what you mean

Comment (nickvergessen) [inline appinfo/remote.php]: fixed

**Comment by RobinMcCorkell** [comment]
Ping - this needs reviving

**Comment by nickvergessen** [comment]
This is blocked/paused by https://github.com/owncloud/calendar/pull/722

**Comment by PVince81** [comment]
Please rebase.

**Comment by nickvergessen** [comment]
Replaced by #775

**Pull request closed without merging**